src/layman/layer/filesystem/gdal.py

```diff
@@ -309,12 +309,18 @@ def normalize_raster_file_async(input_path, crs_id, output_file):
     return process
 
 
-def compress_raster_file_async(*, output_file, file_to_compress, ):
+def compress_and_mask_raster_file_async(*, output_file, input_file_path, band_to_create_mask_from=None):
```

Comment:
Rather use parameter color_interpretation instead of `band_to_create_mask_from`.